.github/workflows/project-assigner.yml

```diff
@@ -0,0 +1,45 @@
+name: Project-assigner
+
+on:
+  pull_request_target:
+    types:
+      - labeled
+      - unlabeled
+  issues:
+    types:
+      - labeled
+      - unlabeled
+
+jobs:
+  assign_to_project:
+    runs-on: ubuntu-latest
+    name: Assign or remove an issue from a project based on a label
+    steps:
```

Comment:
Probably should add an `if` here to only run if it is not already added to the project, in order to not overwrite the status (unless it errors out on its own)